packages/api-fetch/src/index.ts

```diff
@@ -205,4 +205,4 @@ apiFetch.mediaUploadMiddleware = mediaUploadMiddleware;
 apiFetch.createThemePreviewMiddleware = createThemePreviewMiddleware;
 
 export default apiFetch;
-export * from './types';
+export type * from './types';
```

Comment:
This helps esbuild completely eliminate the `./types` import when transpiling. Without this, it imports the `./types` module which is competely empty, and reexports "everything", which turns out to be "nothing". Harmless but useless.